### ISSUE TITLE
Blank output with the setup config in readme

### ISSUE THREAD

**Issue by guoliang** [David-Kunz/gen.nvim#61]
I have tried the following 

```lua
-- Custom Parameters (with defaults)
{
    "David-Kunz/gen.nvim",
    opts = {
        model = "mistral", -- The default model to use.
        display_mode = "float", -- The display mode. Can be "float" or "split".
        show_prompt = false, -- Shows the Prompt submitted to Ollama.
        show_model = false, -- Displays which model you are using at the beginning of your chat session.
        no_auto_close = false, -- Never closes the window automatically.
        init = function(options) pcall(io.popen, "ollama serve > /dev/null 2>&1 &") end,
        -- Function to initialize Ollama
        command = "curl --silent --no-buffer -X POST http://localhost:11434/api/generate -d $body",
        -- The command for the Ollama service. You can use placeholders $prompt, $model and $body (shellescaped).
        -- This can also be a lua function returning a command string, with options as the input parameter.
        -- The executed command must return a JSON object with { response, context }
        -- (context property is optional).
        list_models = '<omitted lua function>', -- Retrieves a list of model names
        debug = false -- Prints errors and the command which is run.
    }
},
```

and 

```lua
require('gen').setup({
  -- same as above
})
```

What seems to work for me was using the following setup given here
https://github.com/David-Kunz/gen.nvim/issues/32#issuecomment-1817778455

However the output is just one line with no line breaks or word wrap.

**Comment by David-Kunz** [comment]
Hi @guoliang ,

Could you try to pin-point what of that configuration is causing the issues?
Could you comment out all lines, until you find the right one?

Thanks and best regards,
David

**Comment by mcginley-s1** [comment]
@David-Kunz  I narrowed this down to 4be150b7421bc0f24c541916502e382666d1d107. Using Ollama locally following the default setup, I checked out that commit, and everything works.  When I checkout 4be150b7421bc0f24c541916502e382666d1d107, aka merge of https://github.com/David-Kunz/gen.nvim/commit/4be150b7421bc0f24c541916502e382666d1d107 then it stops working.  I have to do a few things, but I can do more debugging on that particular PR worst-case tomorrow.
